app/components/meeting-form.js

```diff
@@ -38,12 +38,19 @@ export default class MeetingForm extends Component {
   isSigned = trackedFunction(this, async () => {
     // This is needed here, see https://github.com/NullVoxPopuli/ember-resources/issues/340
     await Promise.resolve();
-    const publishedNotulen = await this.store.query('versioned-notulen', {
-      'filter[zitting][id]': this.zitting.get('id'),
-      'filter[:has:signed-resources]': 'yes',
-      'fields[versioned-notulen]': 'id',
+
+    const signedResources = await this.store.query('signed-resource', {
+      'filter[versioned-notulen][zitting][:id:]': this.zitting.get('id'),
+      'filter[:has-no:deleted]': 'yes',
```

Comment:
this means you will only get signed resources that don't have the "deleted" field
If they have the field but it is false, you won't get them
```
        'filter[:or:][deleted]': false,
        'filter[:or:][:has-no:deleted]': 'yes',
```
afaik this is the way to get both